test/e2e/treatments.e2e.test.ts

```diff
@@ -0,0 +1,92 @@
+import 'dotenv/config.js'
+import * as mwoffliner from '../../src/mwoffliner.lib.js'
+import domino from 'domino'
+import rimraf from 'rimraf'
+import { execa } from 'execa'
+import { jest } from '@jest/globals'
+import { renderers } from './rendererList.js'
+import { zimcheckAvailable, zimdumpAvailable, zimcheck, zimdump } from '../util.js'
+
+jest.setTimeout(200000)
+
+let zimcheckIsAvailable
+let zimdumpIsAvailable
+
+beforeAll(async () => {
+  zimcheckIsAvailable = await zimcheckAvailable()
+  zimdumpIsAvailable = await zimdumpAvailable()
+})
+
+const setDefaultParams = (renderName: string, testId: string) => {
+  return {
+    mwUrl: 'https://en.wikipedia.org',
+    adminEmail: 'test@kiwix.org',
+    outputDirectory: testId,
+    redis: process.env.REDIS,
+    articleList: 'User:Kelson/MWoffliner_CI_reference',
+    forceRender: renderName,
+  }
+}
+
+// Check the integrity of img elements between zim file and article html taken from it
+const verifyImgElements = (imgFilesArr: string[], imgElements: DominoElement[]) => {
+  for (const img of imgElements) {
+    for (const imgFile of imgFilesArr) {
+      if (img.getAttribute('src').includes(imgFile)) {
+        return true
+      }
+    }
+  }
+  return false
+}
+
+const commonTreatmentTest = async (renderer) => {
+  if (!zimcheckIsAvailable) {
+    console.log('Zimcheck not installed, skipping test')
+    return
+  }
+  if (!zimdumpIsAvailable) {
+    console.log('Zimcdump not installed, skipping test')
+    return
+  }
+  const now = new Date()
+  const testId = `mwo-test-${+now}`
+
+  const parameters = setDefaultParams(renderer, testId)
+  await execa('redis-cli flushall', { shell: true })
```

Comment:
I don't know where this is from, but this is very very ugly:
* Because of execa
* Because of flushing everything (maybe other table from other processes)

Reply:
All e2e tests in **test/e2e** folder use **execa** to flush redis store before each test run.